src/com/lftechnology/java_training/jan7/TrainingConstantsUtility.java

```diff
@@ -0,0 +1,15 @@
+package com.lftechnology.java_training.jan7;
+
+/**
+ * it stores the constants values used for assingment jan7
+ * 
+ * @author UtsabNeupane<utsabneupane@lftechnology.com>
+ * 
+ */
+public class TrainingConstantsUtility {
+	// constants initialization
+	static final String path = "/home/utsabn34/codehome/repo/files/";
+	static final String Y = "y";
+	static final String YES = "yes";
```

Comment:
Why is access modifiers not used? I think it should be `public` as this is Constant.